flow/table.go

```diff
@@ -199,6 +199,9 @@ func (ft *Table) expire(expireBefore int64) {
 			}
 
 			logging.GetLogger().Debugf("Expire flow %s Duration %v", f.UUID, duration)
+			f.ReportCounter++
+			f.Status = FlowStatus_ENDED
```

Comment:
could flow have already ended (due to protocol flow termintation), if so then you would not want to handle as timeout